libs/domain/picking/picking-lists-header/picking-lists-header.styles.ts

```diff
@@ -1,6 +1,10 @@
 import { css } from 'lit';
 
 export const pickingListsHeaderComponentStyles = css`
+  :host {
```

Comment:
As for me it should be done in picking lists component, where we are using it. Component itself shouldn't specify it's z-index.